sormas-ui/src/main/java/de/symeda/sormas/ui/user/UserEditForm.java

```diff
@@ -219,11 +219,13 @@ private void updateFieldsByUserRole() {
 		final OptionGroup userRolesField = (OptionGroup)getFieldGroup().getField(UserDto.USER_ROLES);
 
         Set<UserRoleReferenceDto> userRolesFieldValue = (Set<UserRoleReferenceDto>) userRolesField.getValue();
-        final JurisdictionLevel jurisdictionLevel = UserRoleDto.getJurisdictionLevel(userRolesFieldValue.stream().map(userRole -> userRoleMap.get(userRole)).collect(Collectors.toSet()));
+        Set<UserRoleDto> userRoleDtos = userRolesFieldValue.stream().map(userRole -> userRoleMap.get(userRole)).collect(Collectors.toSet());
+        final JurisdictionLevel jurisdictionLevel = UserRoleDto.getJurisdictionLevel(userRoleDtos);
 
-		final boolean hasAssociatedDistrictUser = UserProvider.getCurrent().hasAssociatedDistrictUser();
-		final boolean hasOptionalHealthFacility = UserProvider.getCurrent().hasOptionalHealthFacility();
-		final boolean isPortHealthUser = UserProvider.getCurrent().isPortHealthUser();
+        UserRoleFacade userRoleFacade = FacadeProvider.getUserRoleFacade();
+        final boolean hasAssociatedDistrictUser = userRoleFacade.hasAssociatedDistrictUser(userRoleDtos);
+		final boolean hasOptionalHealthFacility = userRoleFacade.hasOptionalHealthFacility(userRoleDtos);
+		final boolean isPortHealthUser = userRoleFacade.isPortHealthUser(userRoleDtos);
 
```

Comment:
Please align the associate officer dropdown with district: 
![image](https://user-images.githubusercontent.com/47103965/183411084-0e98e78f-109b-4cd6-8f34-95148c0c92dd.png)

Reply:
There is a field (point of entry) in the gap. Basically it would make sense to have only the fields you need in the layout, but since that can change by the choice of user roles, major rebuilding would be needed here. I will not implement this in the context of a bug.